dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentity/TrackerAccessManager.java

```diff
@@ -82,7 +83,7 @@ List<String> canWrite(
 
   List<String> canDelete(UserDetails user, Event event, boolean skipOwnershipCheck);
 
-  List<String> canRead(UserDetails user, Relationship relationship);
+  Set<String> canRead(UserDetails user, Relationship relationship);
```

Comment:
Why do we need to change this to a `Set`?
This whole interface is working with `List`, I would maintain consistency unless there is a pretty important reason

Reply:
When validating relationships, you can have the same error repeating, so i thought it made sense to have a set